governance/votes/Executive vote - October 17, 2024.md

```diff
@@ -0,0 +1,287 @@
+date: 2024-10-17T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Seal Engine Initialization, RWA Vault Debt Ceiling Housekeeping, Pinwheel DAO Resolution, Aave Revenue Share Payment, Spark Proxy Spell - October 17, 2024
+
+The Governance Facilitators, Sidestream, and Dewiz have placed an executive proposal into the voting system. MKR/SKY Holders should vote for this proposal if they support the following alterations to the Sky Protocol.
+
+If you are new to voting in the Sky Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **actions** will occur within the Sky Protocol:
+- The Seal Engine will be initialized. This requires a number of subactions:
+  - Activate a new MKROSM.
+  - Activate a new VoteDelegateFactory contract.
+  - Initialize the Seal Engine farm in the Flapper.
+  - Update Smart Burn Engine parameters.
+  - Initialize the Seal Engine and borrowing facility.
+  - The Multisig responsible for distributing Early Bird Rewards will be funded with newly minted SKY tokens.
+- Debt Ceilings for the now defunct RWA vaults operated by Monetalis will be set to 0.
+- A Pinwheel DAO Resolution with the hash [QmYJUvw5xbAJmJknG2xUKDLe424JSTWQQhbJCnucRRjUv7](https://ipfs.io/ipfs/QmYJUvw5xbAJmJknG2xUKDLe424JSTWQQhbJCnucRRjUv7) will be approved.
+- The Aave-SparkLend Revenue Share payment for Q3 2024 will be distributed.
+- A Spark proxy spell at [0xcc3B9e79261A7064A0f734Cc749A8e3762e0a187](https://etherscan.io/address/0xcc3B9e79261A7064A0f734Cc749A8e3762e0a187) will be executed.
+
+**Voting for this executive proposal will place your MKR/SKY in support of the actions outlined above.**
+
+Unless otherwise noted, the actions listed above are subject to the [GSM Pause Delay](https://sky-atlas.powerhouse.io/#A.1.8.2.1_Pause_Delay-a98b8227-95f6-4711-9d8d-f52cbc6ad2d0|0db30758e055). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Sky Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to [**16 hours**](https://sky-atlas.powerhouse.io/#A.1.8.2.1.2_Pause_Delay_Current_Value-09d2514b-3169-4755-a654-2c774456980d|0db30758e055d2d0).
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. 
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Sky Protocol.
+
+---
+
+## Proposal Details
+
+### Seal Engine Initialization
+
+- **Authorization**: [Governance Poll](https://vote.makerdao.com/polling/QmUm8Krq)
```

Comment:
@MakerJanSky this is missing in the Exec. Sheet

Reply:
Fixed. Thank you, CivicSage.